DMX.Core.Api/Services/Orchestrations/LabOrchestrationService.Exceptions.cs

```diff
@@ -17,6 +18,8 @@ public partial class LabOrchestrationService
     {
         private delegate ValueTask<List<Lab>> ReturningLabsFunctions();
 
+        private delegate ValueTask<Lab> ReturningLabFunctions();
```

Comment:
We should rename this to `ReturningLabFunction` singular. one function at a time.
Same thing goes for the one above it.